src/_guides/language/language-tour.md

```diff
@@ -106,22 +106,14 @@ mind:
     (You might notice when you’re debugging your code that
     variables with no specified type get a special type: `dynamic`.)
     Specifying static types is optional in Dart 1.x, however Dart is moving
-    towards being a fully type safe language. See the next bullet
-    for information on strong mode, which is part of that effort.
+    towards being a fully type safe language.
 
 -   _Strong mode_ contributes to making Dart a type safe language&mdash;you'll
```

Comment:
We can't say that "strong mode is required in Dart 2.0". I got pushback on that. We can't say that it's not optional in Dart 2.0. Also runtime checks actually aren't part of strong mode. Strong mode is part of what makes Dart a sound language, and runtime checks are also part of that.  I addressed the first issue (by not using required) but left the second, because I think it's close enough.